src/TestCaseFilters/GitDirtyTestCaseFilter.php

```diff
@@ -46,7 +49,11 @@ private function loadChangedFiles(): void
         $process = new Process(['git', 'status', '--short', '--', '*.php']);
         $process->run();
 
-        if (! $process->isSuccessful()) {
+        if (!$process->isSuccessful()) {
+            if ($process->getExitCode() === self::EXIT_CODE_GIT_DUBIOUS_OWNERSHIP) {
+                throw new InvalidArgumentException('Dubious folder ownership');
```

Comment:
Perhaps a different exception type would make more sense here. 🤔 `InvalidArgumentException` seems a bit weird possibly.

Reply:
Yeah... I use the generic one because `MissingDependency` inherits from it and I don't know if I could make a new one. I'll change that.

Reply:
Done.